### PR TITLE
Add Definition to contents element

### PR DESCRIPTION
**Pull request by LJBabbage**
### PR Context
Adds a definition element for contents

### Checklist

* [ ] eq-translations updated to support any new schema keys which need translation

### CODE REVIEW

File: schemas/common_definitions.json [hunk 1]
```diff
@@ -63,11 +66,29 @@
...
+      "properties": {
+        "title": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+        },
+        "contents": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+        }
+      }
+    }
+  },
```
Comment (MebinAbraham) [inline schemas/common_definitions.json:91]: Question definitions should be updated to point to this partial as the object is identical.

File: schemas/common_definitions.json [hunk 2]
```diff
@@ -63,11 +66,29 @@
...
+      "type": "object",
+      "required": ["title", "contents"],
+      "properties": {
+        "title": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+        },
+        "contents": {
+          "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+        }
+      }
```
Comment (MebinAbraham) [inline schemas/common_definitions.json:89]: After or before this, you should define: `"additionalProperties": false`

File: schemas/common_definitions.json [hunk 3]
```diff
@@ -63,11 +56,30 @@
...
+    },
+    "additionalProperties": false
```
Comment (MebinAbraham) [inline schemas/common_definitions.json:81]: This is 1 level to deep, it should be directly before or after "properties" object.

Comment (LJBabbage) [reply schemas/common_definitions.json]: I worked that out when my runner PR went bang

**Review by MebinAbraham — APPROVED** [review]

**Review by petechd — APPROVED** [review]

**Pull request merged**